Application/EdFi.Ods.SandboxAdmin/Services/ITemplateDatabaseLeaQuery.cs

```diff
@@ -0,0 +1,16 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Threading.Tasks;
+
+namespace EdFi.Ods.SandboxAdmin.Services
+{
+    public interface ITemplateDatabaseLeaQuery
+    {
+        int[] GetLocalEducationAgencyIds(string sandboxKey);
+
+        Task<int[]> GetLocalEducationAgencyIdsAsync(string sandboxKey);
+    }
```

Comment:
i think this is used in other places and may need to remain in edfi.ods.admin please confirm

Reply:
its used only in EdFi.Ods.Admin and EdFi.Ods.SandboxAdmin.Web